Makefile

```
@@ -57,6 +58,11 @@ proto-lint:
 	@$(DOCKER_BUF) lint --error-format=json
 .PHONY: proto-lint
 
+## proto-check-breaking checks if their is any breaking change to the protos
+proto-check-breaking:
+	@$(DOCKER_BUF) breaking --against $(HTTPS_GIT)#branch=v1.x
```

[question] the first time I ran this, it seems to have timed out. Have you encountered this?

```
$ make proto-check-breaking
Unable to find image 'bufbuild/buf:latest' locally
latest: Pulling from bufbuild/buf
579b34f0a95b: Pull complete
a8989812ba48: Pull complete
cb12522669ac: Pull complete
Digest: sha256:bd804cf64c2d220dd5503d008c49d33cb7a4c6ada9b952170a6f516e45c97531
Status: Downloaded newer image for bufbuild/buf:latest










Failure: the server hosted at that remote is unavailable.
make: *** [proto-check-breaking] Error 1
```

The second time, I see no output... proposal to include a line that informs the developer that the task is actually running.

```
$ make proto-check-breaking
```

```suggestion
proto-check-breaking:
	@echo "--> Checking if Protobuf definitions have any breaking changes"
	@$(DOCKER_BUF) breaking --against $(HTTPS_GIT)#branch=v1.x
```